docs/source/checks/tabular/model_evaluation/plot_weak_segments_performance.py

```diff
@@ -0,0 +1,109 @@
+* `Generate data & model <#generate-data-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+=================================
+
+The check is designed to help you easily identify the model's weakest segments in the data provided. In addition,
+it enables to provide a sublist of the Dataset's features, thus limiting the check to search in
+interesting subspaces.
+
+How Deepchecks automatically detects weak segments
+------------------------------------
+
+The check contains several steps:
+
+#. We calculate loss for each sample in the dataset using the provided model via either log-loss or MSE.
+
+#. Select a subset of features for the the weak segment search. This is done by selecting the features with the highest feature importance to the model provided (within the features selected for check, if limited).
+
+#. We train multiple simple tree based models, each one is trained using exactly two features (out of the ones selected above) to predict the per sample error calculated before.
+
+#. We convert each of the leafs in each of the trees into a segment and calculate the segment's performance.
+
+#. For the model's weakest segments detected we calculate bins for the remaining of the data and calculate the model's
+"""
+#%%
+# Generate data & model
+# =====================
+
+from deepchecks.tabular.datasets.classification.phishing import (
+    load_data, load_fitted_model)
+
+train_dataset, test_dataset = load_data()
+model = load_fitted_model()
+
+#%%
+# Run the check
+# =============
+#
+# The check has several key parameters (that are all optional) that affect the behavior of the
+# check and especially its output.
+#
+# ``columns / ignore_columns``: Controls which columns should be searched for weak segments. By default,
+# a heuristic is used to determine which columns to use based solely on their feature importance.
+#
+# ``alternative_scorer``: Determines the metric to be used as the performance measurement of the model on different
+# segments. It is important to select a metric that is relevant to the data domain and task you are performing.
+# By default, the check uses Neg RMSE for regression tasks and Accuracy for classification tasks.
+#
+# ``segment_minimum_size_ratio``: Determines the minimum size of segments that are of interest. The check is tuned
+# to find the weakest segment regardless of the segment size and so it is recommended to try different configurations
+# of this parameter as larger segments can be of interest even the model performance on them is superior.
+#
+# ``categorical_aggregation_threshold``: By default the check will combine rare categories into a single category called
+# "Other". This parameter determines the frequency threshold for categories to be mapped into to the "other" category.
+#
+# for additional information on the check's parameters, please refer to the API reference of the check
```

Comment:
please put this sentence at the top, and add the link directly (see the train_test_validation quickstart i've previously linked to)